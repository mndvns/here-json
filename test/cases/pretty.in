./bin/here-json -p <<< '{foo: 10, f:[0,3,1]}'
