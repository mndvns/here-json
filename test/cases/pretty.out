{
  "foo": 10,
  "f": [
    0,
    3,
    1
  ]
}